sdk/ai/azure-ai-projects/samples/agents/sample_agents_bing_grounding.py

```diff
@@ -0,0 +1,88 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_bing_grounding.py
+
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the Bing grounding tool from
+    the Azure Agents service using a synchronous client.
+
+USAGE:
+    python sample_agents_bing_grounding.py
+
+    Before running the sample:
+
+    pip install azure.ai.projects azure-identity
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+
+import os
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import BingGroundingTool
+
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
+conn_list = project_client.connections.list()
+conn_id = ""
+for conn in conn_list:
+    if conn.endpoint_url == "https://api.bing.microsoft.com":
+        conn_id = conn.id
+        break
+
+print(conn_id)
+
+# Initialize agent bing tool and add the connection id
+bing = BingGroundingTool()
```

Comment:
add connection also in constructor

For BingGrounding, can there be more than one connection? If not maybe we can remove the add method

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-python/pull/38202/commits/598bfca515f1ae681d9bf278f9d016ab82047d50)